vowpalwabbit/cb_explore_adf.cc

```diff
@@ -109,13 +95,13 @@ example* test_adf_sequence(v_array<example*>& ec_seq)
 }
 
 template <bool is_learn>
-void predict_or_learn_first(cb_explore_adf& data, base_learner& base, v_array<example*>& examples)
+void predict_or_learn_first(cb_explore_adf& data, base_learner& base, multi_ex& examples)
 {
   //Explore tau times, then act according to optimal.
-  if (is_learn && data.gen_cs.known_cost.probability < 1 && test_adf_sequence(data.ec_seq) != nullptr)
-    multiline_learn_or_predict<true>(base, examples, data.offset);
+  if (is_learn && data.gen_cs.known_cost.probability < 1 && test_adf_sequence(examples) != nullptr)
+    base_learn_or_predict<true>(base, examples, data.offset);
```

Comment:
Why isn't this just base.learn(examples)?

Reply:
Changed.